Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -378,7 +379,11 @@ private void startAddCardFlow(@Nullable String cardNumber) {
         getDropInClient().getSupportedCardTypes(new GetSupportedCardTypesCallback() {
             @Override
             public void onResult(List<String> supportedCardTypes, Exception error) {
-                dropInViewModel.setSupportedCardTypes(Arrays.asList(DropInPaymentMethodType.getCardsTypes(supportedCardTypes)));
+                if (supportedCardTypes != null) {
+                    dropInViewModel.setSupportedCardTypes(Arrays.asList(DropInPaymentMethodType.getCardsTypes(supportedCardTypes)));
+                } else if (error != null) {
+                    onError(error);
+                }
```

Comment:
Nitpick - is it more standard / common practice to check the error case first?

Reply:
Yeah we typically do error first - flipped this